lpc11uxx/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+ * Generate using the `--nightly` flag to allow access to overlapping registers
+   (#13)
```

Comment:
Maybe add a note that nightly is required for the crate to work. This way it's not 100% clear.